src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Properties/launchSettings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:51454",
+      "sslPort": 44313
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "api/values",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },   
+    "Altinn.Platform.Authentication": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "api/v1/users",
```

Comment:
For storage, we use something like api/storage/v1/...., i think that was what agreed in last tech forum. @hannekot do you agree?

Reply:
This url is copied from the profile component.. So no apis are defined yet, so maybe I can just remove that line in all?

Reply:
ok, then you can update it when you know the url